### PR TITLE
Increase size of the HGNC symbol field in genes table

### LINKED ISSUE #91: Infinite loop when loading genes in genome build 38
Using the API to load genes in build 38 works, but then the app gets stuck in printing this line:

image

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
-  Fix #91 

### How to test:
- On an instance connected to a real database, load genes in build 38

### Expected outcome:
- [x] Genes should be loaded successfully

### Review:
- [x] Code approved by HS
- [x] Tests executed by CR

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
Test works:

image

File: src/chanjo2/meta/handle_load_intervals.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 LOG = logging.getLogger("uvicorn.access")
 MAX_NR_OF_RECORDS = 10_000
-END_OF_PARSED_FILE = "End of resource file"
+END_OF_PARSED_FILE = "[success]"
```
Comment (henrikstranneheim) [inline src/chanjo2/meta/handle_load_intervals.py:34]:
```suggestion
END_OF_PARSED_FILE: str = "[success]"
```

File: src/chanjo2/meta/handle_load_intervals.py [hunk 2]
```diff
@@ -107,7 +111,7 @@ async def update_genes(
 
 async def update_transcripts(
     build: Builds, session: Session, lines: Optional[Iterator] = None
-) -> int:
+) -> Union[int, None]:
```
Comment (henrikstranneheim) [inline src/chanjo2/meta/handle_load_intervals.py:114]:
```suggestion
) -> Optional[int]:
```

**Review by henrikstranneheim — APPROVED** [review]
👍

**Pull request merged**